src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -319,12 +319,13 @@ export class CreateScenariosComponent implements OnInit {
     // TODO Add error catching for failed scenario creation
     this.planService
       .createScenario(this.formValueToScenario())
-      .subscribe((_) => {
-        // const planId = this.plan$.getValue()?.id;
+      .subscribe((result) => {
+        this.planService.updateStateWithScenario(result.id.toString());
```

Comment:
got pretty confused with this, we have `this.scenarioId` but we are really using the id from `this.planService`....

Reply:
Oh that's a good point — this.scenarioId gets initialized with the id from this.planService, but since we changed the flow that doesn't really make sense anymore (they would create the scenario in the database and get the id before this component would get loaded in). Thanks for fixing this

Reply:
Oh that's a good point — this.scenarioId gets initialized with the id from this.planService, but since we changed the flow that doesn't really make sense anymore (they would create the scenario in the database and get the id before this component would get loaded in). Thanks for fixing this